apps/public-docsite/src/pages/Styles/FabricIconsPage/docs/web/FabricIconsOverview.md

```diff
@@ -1,7 +1,7 @@
-Fluent UI uses a custom font for its iconography. This font contains glyphs that you can scale, color, and style in any way. There are also mirrored icons available for right-to-left localization. You can also download and install the icon font to use it with various design apps like Sketch, Figma, or Adobe XD.
+Fluent UI primarily uses a custom font for its iconography, released under the [Microsoft Fabric Assets License](https://aka.ms/fluentui-assets-license). As of Fluent UI React version 8, an SVG-based version of the same icon set is available under the MIT license.
```

Comment:
Maybe say "in the past, Fluent UI has used a custom font", but as of Fluent UI 8 it uses SVG?

Reply:
> Fluent UI has traditionally used a custom font for its iconography. This font is part of the MDL2 design system and contains glyphs that you can scale, color, and style in any way. You can also download and install the [icon font](#/resources) to use it with various design apps like Sketch, Figma, or Adobe XD.

like this line from below. 'traditionally' works

Reply:
v8 still uses a font by default, the difference is that we also have SVG available